cloud/pkg/cloudstream/streamserver.go

```diff
@@ -136,6 +136,7 @@ func (s *StreamServer) getContainerLogs(r *restful.Request, w *restful.Response)
 		session:      session,
 		ctx:          r.Request.Context(),
 		edgePeerStop: make(chan struct{}),
```

Comment:
Need to set the size to `2`?

Reply:
As we discussed, edgePeerStop channel of log_connnecton and metrics_connnection will not block the sender side.